### ISSUE TITLE
Rendering of "NIST IR 8202" citation seems to lack details #190

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-ietf#191]
Originally from @manuelfuenmayor  https://github.com/relaton/relaton-nist/issues/107

# Description

This markup:
```
* [[[NISTIR_8202,NIST IR 8202]]]
```

renders in TXT as:

```
   [NISTIR_8202]
              , , , and , "Blockchain technology overview",
              NIST IR 8202, DOI 10.6028/NIST.IR.8202, October 2018,
              <https://doi.org/10.6028/NIST.IR.8202>.
```

in HTML:
![Screenshot 2023-10-19 215354](https://github.com/relaton/relaton-nist/assets/33627611/18e23aa3-5dbd-42e0-b240-6bea51bc620e)

A preferred result would be:
![Screenshot 2023-10-19 215455](https://github.com/relaton/relaton-nist/assets/33627611/53ca3cc7-e9d3-4b37-8128-70c0e2e20644)

# To reproduce

You can compile the full document in https://github.com/cisba/i-d_dlts/pull/1, or use
this little sample: [nist-ref.zip](https://github.com/relaton/relaton-nist/files/13049223/nist-ref.zip)

**Comment by ronaldtse** [comment]
As seen in https://github.com/relaton/relaton-nist/issues/107#issuecomment-1778456828 , Relaton does provide all those names in the record. So this is a Metanorma-IETF citation rendering issue.

**Comment by opoudjis** [comment]
````
  [NISTIR_8202]
              Yaga, D., Mell, P., Roby, N., and K. Scarfone, "Blockchain
              technology overview", NIST IR 8202,
              DOI 10.6028/NIST.IR.8202, October 2018,
              <https://doi.org/10.6028/NIST.IR.8202>.
````

**Comment by opoudjis** [comment]
Fixed in #190

**Comment by ronaldtse** [comment]
This is fully fixed, I've confirmed.

<img width="741" alt="Screenshot 2023-10-31 at 1 31 26 PM" src="https://github.com/metanorma/metanorma-ietf/assets/11865/96980569-cf19-4ef4-bac5-516fb7f11df6">
